lib/Sema/TypeCheckMacros.cpp

```diff
@@ -456,6 +456,13 @@ static std::string adjustMacroExpansionBufferName(StringRef name) {
   return result;
 }
 
+Optional<unsigned>
+ExpandMacroExpansionExprRequest::evaluate(Evaluator &evaluator,
+                                          MacroExpansionExpr *mee) const {
+  assert(mee->getMacroRef() && "MacroRef should be set before expansion");
+  return expandMacroExpr(mee);
```

Comment:
Can we remove `expandMacroExpr` and move its implementation into here? All callers should be using the request right?